pages/linux/picom.md

```diff
@@ -0,0 +1,16 @@
+# picom
+
+> Standalone compositor for Xorg.
+> More information: <https://wiki.archlinux.org/title/picom>.
+
+- Enable picom during a session:
+
+`picom &`
+
+- Start picom as a background process:
+
+`picom &`
```

Comment:
I think the above example and this one is exactly same, can we probably club them together :)

Reply:
My bad, sorry! These should be two different commands
`picom &` and `picom -b`. For some reason I kept them the same. Fixing that right away